### PR TITLE
Proposed changes - Accessing the System, Good Citizenship, File Management

### PR DESCRIPTION
**Pull request by lhelms2**
@pmenstrom, @craigsteffen, @wjackson-gh

Accessing the System, Good Citizenship, and File Management sections have been revised and are ready for the review/merge process. 
Once these are updates are merged, I’ll move onto another set of sections.

Thank you,
Lillian

### CODE REVIEW

**Review by pmenstrom — APPROVED** [review]
I read through the text and the information looks good.

**Comment by craigsteffen** [comment]
I looked through the editorial changes.  The formatting and organizational changes are all fine.  The content is all fine.  

It looks like, though, there's an old stale notion of how the interactive nodes are named that I somehow left in here and didn't update.  My ssh examples seem to claim that nodes are named for allocation groups, which I don't think is true.  I'm going to update those sections in this branch right now.  

I think after that's done this can get merged into main as far as I"m concerned.  I'll comment here when it's done.

**Comment by craigsteffen** [comment]
Yeah, I think I want to modify the ssh instructions.  I'll work on that right now, and plan to have a commit in by noon central.

File: docs/source/user_guide/file_mgmt.rst [hunk 1]
```diff
@@ -29,145 +29,186 @@ The table below describes the location, purpose, and quota policy for the storag
...
 The /dataset area contains curated, read-only datasets typically exported from a Postgres database. Projects will be informed when data is placed here for them.
 
 Transferring Files
 -------------------
 
-Copying project data to your home directory
+Copying Project Data to Your Home Directory
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You will not work directly on your group's curated project data in the /datasets directory. Instead, you will need to copy the data from the original directory to your home directory.  Follow the steps below.
```
Comment (pmenstrom) [inline docs/source/user_guide/file_mgmt.rst:69]: They could also copy the data to scratch or their project space.

**Review by pmenstrom — APPROVED** [review]

File: docs/source/user_guide/file_mgmt.rst (hunk 1 above)
Comment (lhelms2) [reply docs/source/user_guide/file_mgmt.rst]: I updated this section to include scratch and projects in addition to home.

**Pull request merged**